src/app/code/community/Shopgate/Framework/Model/Carrier/Fix.php

```diff
@@ -90,7 +90,14 @@ protected function createSgShippingMethod(ShopgateCartBase $shopgateOrder)
         $method->setData('carrier', $this->_code);
         $method->setData('carrier_title', $group !== ShopgateDeliveryNote::OTHER ? $group : '');
         $method->setData('method', $this->_method);
-        $method->setData('method_title', $shippingInfo->getDisplayName());
+        if ($shopgateOrder->getShippingType()
+            == Shopgate_Framework_Model_Shopgate_Shipping_Mapper::SHIPPING_TYPE_PLUGINAPI
```

Comment:
strict `===` sign is preferred, 
Also shorten it up, it's bit too verbose and old school -> `? :`